.babelrc

```diff
@@ -6,13 +6,13 @@
         "modules": false,
         "useBuiltIns": "entry",
         "core-js": 3,
+        "shippedProposals": true,
       }
     ],
     "@babel/preset-react"
   ],
   "plugins": [
     ["styled-components", { "displayName": true }],
-    "@babel/plugin-proposal-class-properties"
```

Comment:
Honestly didn't check, but just ruled it out of my mind since it was behaving differently between `yarn` and `npm` at that point after I initially did the `yarn upgrade-interactive`.

Reply:
Went back to the commit and checked; this was not the problem. The problem was we had multiple versions of babel installed, and I believe an older one was still being used by parcel at that time.

Reply:
what is shippedProposals?

Reply:
@rperez89 that’s an option of `@babel/preset-env` to enable ES features that are shipped in browsers, regardless of their TC39 stage: https://github.com/babel/website/blob/0d482dd7ed3451592549009483423f7d37d2fe92/docs/preset-env.md#shippedproposals